<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackWidget</class>
 <widget class="QWidget" name="TrackWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>124</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0,0,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="HttpImageWidget" name="albumArt">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="trackTitleFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="trackTitleLayout" stretch="0,0,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="unicorn::Label" name="trackTitle">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string notr="true">Title</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="asterisk">
          <property name="text">
           <string notr="true">*</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="unicorn::Label" name="artist">
       <property name="text">
        <string notr="true">Artist</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="equaliser">
         <property name="text">
          <string notr="true">e</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="unicorn::Label" name="timestamp">
         <property name="text">
          <string notr="true">Timestamp</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="love">
     <property name="text">
      <string>Love</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="tag">
     <property name="text">
      <string>Tag</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="share">
     <property name="text">
      <string>Share</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buy">
     <property name="text">
      <string>Buy</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>unicorn::Label</class>
   <extends>QLabel</extends>
   <header>lib/unicorn/widgets/Label.h</header>
  </customwidget>
  <customwidget>
   <class>HttpImageWidget</class>
   <extends>QLabel</extends>
   <header location="global">lib/unicorn/widgets/HttpImageWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
